RFC-003-set-up-google-account.md

```diff
@@ -0,0 +1,169 @@
+
+For that reason, when the account is first created, the account credentials will be shared with:
+
+- The current project chair (Alyssa)
+- The vice-chair (we need to create this position and nominate someone to it)
+- The templates coordinator (we need to create this position and nominate someone to it; a future RFC that I'm submitting will make a recommendation for this role and it will likely be Aidan Doherty)
+
+Other individuals may possibly request access to the Google account for these reasons:
+
+- You are leading multiple working groups and need to update meeting details regularly
+- Your working group is creating a lot of files and assets that should be stored in the shared project Google Drive
+- You have been authorized by the project steering committee to create an account with another service using our Google account
+- You need to use the project mail account regularly to communicate with others in an official capacity
+- Additional important reasons that we haven't yet thought of
+
+Members who are granted Google account access should be trusted by the community.
+They should have belonged to and/or contributed to the community for at least a quarter (3 months).
+Being a member of the project steering committee helps but should not necessarily be required.
+
+Access can be granted by reaching out to the project chair or vice-chair, who will raise a motion and encourage voting from the project steering committee members.
+
+
+### Rotating passwords
+
+I propose we set up a cadence for changing our account passwords.
+Unless someone thinks it should be more frequent, I'm open to changing the password once a year.
+We could also consider changing it when certain project events occur, such as when a PSC member who had access steps down.
+
+
+### Why a free Google account as opposed to something else?
+
+You might wonder why I'm propsing that we set up a free, basic Google/Gmail account as opposed to some other service.
+
+I researched the prospect of creating a Google Workspaces (formerly known as GSuite) account for our project, but since we are not a 501(c)(3), we would not qualify for a free account.
+We could pay for an account (it's roughly $12-18 /user/month), but then we'd introduce the complications of taking in funds and paying for accounts.
+Our project probably isn't ready for financial complexity yet, especially since we don't have a formal legal status or an acting treasurer.
+
+I'm not aware of any other viable options that give us the functionality that we need, but I'm open to suggestions.
+Carrie Crowe also researched other options and reached the same conclusion that I reached.
+
+
+### Risks or reasons not to adopt this RFC
+
+The only risks I can think of are that if we make our community calendar public, we could increase the chance of bad actors crashing our meetings, but I think this is a risk worth taking.

```

Comment:
Suggestion: I think we have a few risks:

```suggestion
The risks I can think of are:
* If we make our community calendar public, we could increase the chance of bad actors crashing our meetings, but I think this is a risk worth taking.
* A disgruntle community member might delete all docs in our shared repository. We can mitigate this by moving docs into github once they become mature, and reduce the number of community members who know the project password.
* Community members might put personal or sensitive information into a publicly visible repository.

```

Reply:
I'm not going to commit your suggestion, but I've worked it all into my latest commit. The only reason I didn't fully adopt it is because I wanted to tweak the wording a little bit.

Reply:
two cents on the 'crashing meetings' ... we had this happen to a Season of Docs Meet and it was extremely unpleasant. There are coordinated groups who look for open links and bomb them with attack sounds, NSFW content, and more. Once the trolls joined, we were unable to keep them from sharing their screens and we could not keep them muted. There are some new features in Meet that may mitigate this, but wanted to share this fwiw.

Reply:
@emckean , that's great feedback. What do you recommend as a workaround or preventative measure?

Reply:
I think that we could have the calendar public, but not allow autojoining (eg not allow "anyone with the link" to join), and both require that anyone who wants to join ping 24 hours in advance of the meeting and have a policy of not allowing unknown users (eg not already active members or people who pinged to join in advance) into the meeting during the meeting.

Reply:
I've incorporated Erin's comments into the final version of this RFC. Resolving this conversation.